server/views/components/image/image.njk

```diff
@@ -1,4 +1,7 @@
 {% set sizes = model | getImageSizesFor %}
+{% set useIiif = featuresCohort | isFlagEnabled('useIiif', featureFlags) %}
+{% set useIiifOrigin = featuresCohort | isFlagEnabled('imagesFromOrigin', featureFlags) %}
```

Comment:
I can't really see why we would have both of these?
Surely we should never be using the origin?

Reply:
that's there at request of platform team #1289